pkgs/by-name/co/cosmic-icons/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, just
+, pop-icon-theme
+, hicolor-icon-theme
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "cosmic-icons";
+  version = "unstable-2023-08-30";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = pname;
+    rev = "14d8e2048087be1ad444f9b3ebb75885509f72c6";
+    sha256 = "sha256-WbdgHmTn403x95x9wEYL0T9ksbN+YLzEB2yE0UrF9T0=";
+  };
+
+  nativeBuildInputs = [ just ];
+
+  justFlags = [
+    "--set"
+    "prefix"
+    (placeholder "out")
+  ];
+
+  propagatedBuildInputs = [
+    pop-icon-theme
+    hicolor-icon-theme
+  ];
+
+  dontDropIconThemeCache = true;
+
+  meta = with lib; {
+    description = "System76 Cosmic icon theme for Linux";
+    homepage = "https://github.com/pop-os/cosmic-icons";
+    license = with licenses; [
+      cc-by-sa-40
+      gpl3
+    ];
+    platforms = platforms.linux;
```

Comment:
Any reason? Doesn't seem like something that would be platform restricted.

Reply:
I wasn't sure if it would work under darwin, since I can't test it. Should I just open it up anyway?

Reply:
`platforms` should be the set of platforms a package _should_ work on.